it/it-tests/src/test/java/it/authorisation/ExecuteAnalysisPermissionTest.java

```diff
@@ -89,28 +91,44 @@ public void no_need_for_browse_permission_to_scan() throws Exception {
     runProjectAnalysis(orchestrator, "shared/xoo-sample", "sonar.login", USER_LOGIN, "sonar.password", "thewhite");
 
     // Remove browse permission for groups Anyone on the project
-    removeGroupPermission("anyone", "sample", "user");
+    removeProjectPermission("anyone", "sample", "user");
 
     // still no error
     runProjectAnalysis(orchestrator, "shared/xoo-sample", "sonar.login", USER_LOGIN, "sonar.password", "thewhite");
   }
 
-  private static void addPermission(String groupName, String permission) {
+  @Test
+  public void execute_analysis_permission_only_on_project() throws Exception {
+    removeGlobalPermission("anyone", "scan");
+    addProjectPermission("anyone", PROJECT_KEY, "scan");
+    addGlobalPermission("anyone", "dryRunScan");
```

Comment:
dryRunScan is the 'Preview Analysis' permission. It's now required (in order to have access to the batch/global WS)